src/averageAges.js

```diff
@@ -14,12 +19,17 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menAgeSum = people
+    .filter(person =>
+      century
+        ? Math.ceil(person.died / 100) === century
+        && person.sex === 'm'
+        : person.sex === 'm'
```

Comment:
the ternary operator is easier to read when there are no additional "elements" between the `?` and the `:`
```suggestion
        ? Math.ceil(person.died / 100) === century && person.sex === 'm'
        : person.sex === 'm'
        
        // or make some indentation to show this as one part
        ? (Math.ceil(person.died / 100) === century
          && person.sex === 'm')
        : person.sex === 'm'
```